packages/test/test-end-to-end-tests/src/test/tsconfig.json

```diff
@@ -4,10 +4,10 @@
 		"declaration": false,
 		"declarationMap": false,
 		"rootDir": "../",
-		"outDir": "../../dist",
+		"outDir": "../../lib",
```

Comment:
will this impact "Debug Current Mocha Test" in the launch.json?

Reply:
yes, good catch. I'm hesitant to extend the launch config to make things work (will involve making VSCode load more files which was a big contributor to the launch config taking a long time). so perhaps I'll just document that it'd be good to make this more consistent in the future.